README.md

```diff
@@ -46,8 +46,15 @@ A command is required
 npm install @usedispatch/client
```

Comment:
Lets split the doc into dispatch mailbox and dispatch postbox.

And add the npm installs in them independently.